kustomize/commands/internal/util/util_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2019 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 
-package util
+package util_test
```

Comment:
Are there guidelines for naming a test file in a package? Should we always use `<package-name>_test`?  I see inconsistent usage across the kustomize repository.

Reply:
Yes, we have a linter that checks that test packages have the suffix `_test`. I've been hit by that a few times and made the change proactively here.

Example run where linting failed because of that: https://github.com/kubernetes-sigs/kustomize/actions/runs/6647723299

Reply:
Thanks. I find it strange that the linter did not complain about the package name for [configmap_test.go](https://github.com/kubernetes-sigs/kustomize/pull/5430/files#diff-b20357d114ddb6a9017a3ff759b5d1d3adc38c96e00aef632a76c3eec34d35e9L4) and [secret_test.go](https://github.com/kubernetes-sigs/kustomize/pull/5430/files#diff-43233ee2c7ddb4c285c911303178deaf4bf8274355e66705bd26ad08bc512997L4). Is the linter not checking those directories? Should we update those package names for consistency?

Reply:
This is indeed strange 👀 maybe it's a flaky check. I can update those later.

Reply:
Updated for consistency!